### DISCUSSION TITLE
Choppy sped up recordings. Go2rtc stream is smooth though.

### DISCUSSION THREAD

**Discussion by rb2k** [blakeblackshear/frigate#8286] [Ask A Question] [answered]
A bit of an odd problem.
I use Frigate (0.13 b1) with go2rtc 'proxying' all streams from a reolink camera.
I also have a test install of "Blue Iris" set up that is pointing at the go2rtc streams that frigate is providing as well.

So in general, frigate and blue iris use the same video stream as input.

The odd thing:
For the same event, Frigate has it listed as 18 seconds, but the export/recording is only 1 second long and super sped up.
Blue Iris has a 'normal' video.

FFProbe for the frigate file says:
```
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from '/Users/marc/Downloads/frigate.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    encoder         : Lavf59.27.100
  Duration: 00:00:00.76, start: 0.000000, bitrate: 72818 kb/s
  Stream #0:0[0x1](und): Video: h264 (High) (avc1 / 0x31637661), yuv420p(progressive), 2560x1440, 72795 kb/s, 348.22 fps, 15 tbr, 15360 tbn (default)
    Metadata:
      handler_name    : VideoHandler
      vendor_id       : [0][0][0][0]
```

I'm a bit confused about the 348.22 fps part.

Any ideas what could cause this? It's a pretty regular occurence.

**Comment by NickM-27** [comment]
What is your config

**Reply by rb2k** [reply]
The relevant parts are probably these:
```
ffmpeg:
  # Optional: global ffmpeg args (default: shown below)
  global_args:
    - -hide_banner
    - -loglevel
    - error
  hwaccel_args: preset-vaapi

[...]  

    sqwrlcam_frigate:
      - http://X/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=********&password=*******
      - "ffmpeg:sqwrlcam_frigate#audio=opus"
      [...]
     sqwrlcam_sub:
      - http://X/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=********&password=**********


[...]     
     
 
  sqwrlcam_frigate:
    ffmpeg:
      inputs:
        - path: rtsp://127.0.0.1:8554/sqwrlcam_frigate?video=copy&audio=aac
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://127.0.0.1:8554/sqwrlcam_sub?video=copy
          input_args: preset-rtsp-restream
          roles:
            - detect
    detect:
      fps: 7
      height: 360
      width: 640
    motion:
      mask:
        - 0,0,454,0,512,0,640,0,640,177,0,118
    record:
      events:
	retain:
          mode: motion
```

I wonder if it's related to this somewhat older thread: https://github.com/blakeblackshear/frigate/issues/1467

**Reply by NickM-27** [reply]
> I wonder if it's related to this somewhat older thread: #1467

No, you're not using http flv in frigate. Also reolink is very common in frigate and others (including myself) are not seeing this issue.

**Reply by NickM-27** [reply]
Actually you're not following the recommended reolink config https://deploy-preview-6262--frigate-docs.netlify.app/configuration/camera_specific#reolink-cameras

**Reply by rb2k** [reply]
II don't think I have set it to timelapse. The video in the web UI plays the same way as the export.
I don't think I have set/selected a timelapse anywhere.

I think this is the recommended config? http stream on port 1935 from the camera with a video copy and then preset-rtsp-restream from go2rtc?

**Reply by NickM-27** [reply, marked as answer]
> I think this is the recommended config? http stream on port 1935 from the camera with a video copy and then preset-rtsp-restream from go2rtc?

no, you are not using ffmpeg for the entire thing, only for audio, which is not what is recommended for issues exactly like this because reolink sends bad timestamps

**Reply by rb2k** [reply]
Ohhhhh, I think I misunderstood the notation. Thanks for pointing that out, I will switch it over and see what happens.

Still a bit odd that Blue Iris was able to somehow deal with the go2rtc stream with the existing config

**Comment by rb2k** [comment]
https://github.com/blakeblackshear/frigate/assets/9519/fa2011df-7a8f-438f-8915-baba8b6dc99e

Here's the [Frigate export](https://github.com/blakeblackshear/frigate/assets/9519/fa2011df-7a8f-438f-8915-baba8b6dc99e)

**Reply by NickM-27** [reply]
Looks like you selected time lapse as the export type

**Reply by rb2k** [reply]
Oddly no. This only happens every now and then (10% of the time if I had to guess). Other videos from the same camera are fine.
